internal/providers/terraform/azure/testdata/storage_account_test/storage_account_test.golden

```diff
@@ -89,56 +89,153 @@
+ ├─ Capacity                                               1,000,000  GB                          $15,200.00 
+ ├─ Write operations                                             100  10K operations                  $10.00 
+ ├─ List and create container operations                         100  10K operations                   $5.50 
+ ├─ Read operations                                               10  10K operations                   $0.10 
+ ├─ All other operations                                         100  10K operations                   $0.44 
+ ├─ Data retrieval                                             1,000  GB                              $10.00 
+ ├─ Blob index                                                    10  10K tags                         $0.39 
+ └─ Early deletion                                             1,000  GB                              $15.20 
+                                                                                                             
+ azurerm_storage_account.v2_cool_lrs_nfsv3                                                                   
+ ├─ Capacity                                               1,000,000  GB                          $15,000.00 
+ ├─ Iterative Write operations                                 1,700  100 operations                 $221.00 
+ ├─ Write operations                                             100  10K operations                  $13.00 
+ ├─ Iterative Read operations                                     15  10K operations                   $1.08 
+ ├─ Read operations                                               10  10K operations                   $0.13 
+ ├─ All other operations                                         100  10K operations                   $0.60 
+ ├─ Data retrieval                                             1,000  GB                              $10.00 
+ └─ Early deletion                                             1,000  GB                              $15.00 
+                                                                                                             
+ azurerm_storage_account.v2_cool_ragzrs                                                                      
+ ├─ Read operations                                               10  10K operations                   $0.10 
+ ├─ All other operations                                         100  10K operations                   $0.44 
+ └─ Data retrieval                                             1,000  GB                              $10.00 
+                                                                                                             
+ azurerm_storage_account.v2_hot_gzrs                                                                         
+ ├─ Read operations                                               10  10K operations                   $0.04 
+ └─ All other operations                                         100  10K operations                   $0.44 
```

Comment:
Why are these resources not showing pricing for all cost components?

Reply:
For `StorageV2` the GZRS/RA-GZRS redundancy types are inconsistent for different regions. Sometimes even  common types don't have prices for common regions, for example, `westus` - "Write operations" is missing for V2 Hot ZRS.

This is why I couldn't get rid of `IgnoreIfMissingPrice` flag. If price is missing for this region+redundancy type then it is not shown. I decided to show everything that can be found for every permutation instead of ignoring the resource completely.